src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -36,6 +36,48 @@
  * <pre>
  * &lt;module name=&quot;InnerTypeLast&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test { // violation, field should be declared before inner types.
```

Comment:
violation comment is misplaced.

comments should be on lines where violation is reported.

Reply:
Done